Source/bmalloc/libpas/src/libpas/pas_lock.h

```diff
@@ -124,6 +124,14 @@ PAS_END_EXTERN_C;
 #define OS_UNFAIR_LOCK_INLINE 1
 #endif
 #include <os/lock_private.h>
+
+// Swift Cpp Interop cannot import the inline variants.
+#if defined(__swift__)
+#define os_unfair_lock_lock_with_options_inline os_unfair_lock_lock_with_options
+#define os_unfair_lock_trylock_inline os_unfair_lock_trylock
+#define os_unfair_lock_unlock_inline os_unfair_lock_unlock
+#endif
+
```

Comment:
I worry if this is an ODR violation which could cause performance regressions. These functions may be used to generate multiple copies of the same instantiation in different files, and at link time, a copy generated for Swift compilation may be chosen. Then all paths through that template instantiation will use the slow path.

Reply:
fixed this by passing in OS_UNFAIR_LOCK_INLINE=1